lib/langchain/llm/response/google_gemini_response.rb

```diff
@@ -27,7 +27,11 @@ def embedding
     end
 
     def embeddings
-      [raw_response.dig("predictions", 0, "embeddings", "values")]
+      if raw_response.key?("embedding")
```

Comment:
Had to do something weird here since the GoogleGeminiResponse class was also used by GoogleVertexAI and the response of the embeddings is slightly different, see `spec/fixtures/llm/google_gemini/embed.json` below...